File: alphabit-coding-challenge-03/gunpower/solutions/gunpower.py
TAB=[]

N,B=[int(s) for s in input().split(" ")]

TAB = [int(x) for x in input().split(" ")] 
count=0

#or you make a loop for sorting the table

TAB.sort()

for i in range(len(TAB)):

    if(B>=TAB[i]):

        B=B-TAB[i]

        count+=1

print(count)

File: alphabit-coding-challenge-03/gunpower/solutions/gunpower.java
#include <iostream>
#include <bits/stdc++.h>
#include <math.h>

#define max 100000
using namespace std;

int main()
{
    int n, b,  count,i ;
    int tab[max];
    //input*************************************

 cin >>  n >> b;
	for (i = 0; i < n; i++) {
	cin >> tab[i];
	}



    // traitment*************************************
		sort(tab,tab+n);//,sizeof(int),cmpfunc);

    count = 0;
	for (i = 0; i < n; i++) {
		if (b >= tab[i]) {
			b -= tab[i];
			count++;
		}
	}

    //output*************************************

 cout <<  count;



    return 0;
}